### PR TITLE
Prepare changelog for upcoming release

### PR DESCRIPTION
**Pull request by lbonn**
Looks like we don't have anything to get from the docs branch this time

### CODE REVIEW

File: docs/README.adoc [hunk 1]
```diff
@@ -38,6 +38,7 @@ The link above is for the doxygen docs on master. Doxygen docs for the following
 * https://advancedtelematic.github.io/aktualizr/2020.4/index.html[2020.4]
 * https://advancedtelematic.github.io/aktualizr/2020.5/index.html[2020.5]
 * https://advancedtelematic.github.io/aktualizr/2020.6/index.html[2020.6]
+* https://advancedtelematic.github.io/aktualizr/2020.7/index.html[2020.7]
```
Comment (patrickvacek) [inline docs/README.adoc:41]: You should update `docs/ota-client-guide/modules/ROOT/pages/_partials/aktualizr-version.adoc` (and `docs/ota-client-guide/antora.yml`?) as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
...
+
+### Changed
+
+- Cache device data (network, hardware info...) as much as we can to save bandwidth [PR](https://github.com/advancedtelematic/aktualizr/pull/1673)
+
+### Fixed
+
+- Various docker-app fixes: [PR](https://github.com/advancedtelematic/aktualizr/pull/1664) [PR](https://github.com/advancedtelematic/aktualizr/pull/1665)
+- Use ED25519 to sign manifests when set as key type: [PR](https://github.com/advancedtelematic/aktualizr/pull/1608)
+
```
Comment (patrickvacek) [inline CHANGELOG.md:20]: Maybe add something about changing the installed Target matching logic from https://github.com/advancedtelematic/aktualizr/pull/1666?

Comment (lbonn) [reply CHANGELOG.md:20]: Do you have a suggestion to formulate this change? "Stricter checks for uptane metadata consistency"?

Comment (patrickvacek) [reply CHANGELOG.md:20]: How about "Stricter matching of Uptane metadata with installed images."?

Although to be honest, it's just that the strict check was moved from the install step to the check for updates.

Comment (lbonn) [reply CHANGELOG.md:21]: Ok changed it to yours

**Pull request merged**